rules/S7022/secrets/metadata.json

```
@@ -0,0 +1,56 @@
+{
+  "title": "SECRET_TYPE should not be disclosed",
```

You forgot to change the rule title.
```suggestion
  "title": "Azure Machine Learning keys should not be disclosed",
```